### PR TITLE
Validate null property value for Common Schema

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/ingestion/models/one/PartCUtilsAndroidTest.java [hunk 1]
```diff
@@ -40,6 +40,14 @@ public void nullKey() {
         PartCUtils.addPartCFromLog(properties, log);
     }
 
+    @Test(expected = IllegalArgumentException.class)
```
Comment (jaelim-ms) [inline sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/ingestion/models/one/PartCUtilsAndroidTest.java]: Can you also add a test with empty string value?

Comment (guperrot) [reply sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/ingestion/models/one/PartCUtilsAndroidTest.java]: Done
